src/components/Charts/ChataChart/ChataChart.js

```diff
@@ -192,9 +191,35 @@ export default class ChataChart extends Component {
     return { chartHeight, chartWidth }
   }
 
+  sortChartDataByDate = (props) => {
+    try {
+      if (!props.data || typeof props.data !== 'object') {
+        return undefined
+      }
+      const dateColumnIndex = getDateColumnIndex(props.columns)
```

Comment:
There may be more than 1 date column so "getDateColumnIndex" might not always find the right one (I think it just picks the first one)

Instead you could just grab props.stringColumnIndex since you already checked that it was a date type column below